azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -223,8 +225,9 @@ def get_processes_in_agent_cgroup(self):
             The return value can be None if cgroups are not enabled or if an error occurs during the operation.
             """
             def __impl():
-                agent_unit = self._cgroups_api.get_agent_unit_name()
-                return self._cgroups_api.get_processes_in_cgroup(agent_unit)
+                if self._agent_cpu_cgroup_path is None:
```

Comment:
Wouldn't it be better to use the memory cgroup here since we know CPU is not mounted by default in some distros, whereas memory is?

Reply:
No, it is CPU that we are interested in.

Reply:
Why CPU specifically? Aren't we only using the cgroup path to get the PIDs? They are also stored in the memory cgroup.

Reply:
We want to enforce CPU, so it is the CPU cgroup that we need to check.